projects/sensory_interface/index.html

```diff
@@ -9,9 +9,18 @@
 <body>
     <textarea id="textInput">1	2	3
 4	5	6</textarea>
-    <button id=" processButton" onclick="processData()">process </button>
+    <button onclick="processData()">process </button>
     <br>
     <div id="tableContainer"></div>
+    <!-- The core Firebase JS SDK is always required and must be listed first -->
```

Comment:
Right now you're using only Firebase hosting, and I don't think you need the scripts for that.
Try removing them and deploying and see if it still works.

Reply:
it works without the scripts as you suggested. Removing...

Reply:
Great :)